scroll-animations-1/Overview.bs

```diff
@@ -1219,6 +1219,15 @@ run), the user agent may likewise choose not to sample scroll-driven animations
 for that composited frame. In such cases, the rendered scroll offset and the
 state of a scroll-driven animation may be inconsistent in the composited frame.
 
+<h2 id="predefined-scroll-timelines">
+Predefined Scroll Timelines</h2>
+
+  The following stylesheet fragments provide the normative definition 
+  for predefined scroll timelines that can be used by authors:
+
+  <pre class='stylesheet'><bdo dir=ltr>
+    @scroll-timeline document-timeline {}
```

Comment:
document-timeline may not be a great name, it sounds a lot like https://www.w3.org/TR/web-animations-1/#the-documents-default-timeline which it's not. What would we call this if we were to add it to the Document object alongside the DocumentTimeline `document.timeline`?

Reply:
i updated to be more specific: `document-scroll-timeline`, which could be `document.scrollTimeline` in the DOM. thoughts?